### PR TITLE
Gha build

### PR DESCRIPTION
**Pull request by tonyskapunk**
Add GitHub Action for pushing new builds

This depends on #1 to build successfully the images

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	@if [[ ! -d testpmd/testpmd-as-load-balancer ]]; then \
+	git clone $(TESTPMD_LB_REPO) testpmd/testpmd-as-load-balancer; \
+	fi
+
+.PHONY: build-all # Build ALL images
+build-all: build-testpmd build-mac build-listener
+
+.PHONY: build-testpmd # Build testpmd
+build-testpmd: testpmd-dependencies
+	@$(CONTAINER_CLI) build $(@:build-%=%) -f $(@:build-%=%)/Dockerfile -t "$(REGISTRY)/$(ORG)/testpmd-container-app-$(@:build-%=%):$(TAG)" $(CONTAINER_ARGS)
```
Comment (betoredhat) [inline Makefile:32]: I think you will also need to use centos stream here: https://github.com/rh-nfv-int/testpmd-container-app/blob/master/testpmd/Dockerfile

Comment (tonyskapunk) [reply Makefile:32]: Yup, See #1

**Review by betoredhat — APPROVED** [review]
LGTM, thanks

**Review by betoredhat — APPROVED** [review]
LGTM

**Pull request merged**